docs/README.md

```diff
@@ -19,4 +19,4 @@ Beta-releases can be obtained by opting in to the [Beta testing channel](https:/
 # How do I learn more?
 Please see the [documentation](Documentation.md).
 
-[![Build Status](https://www.bitrise.io/app/43a23ab54dee9f7e/status.svg?token=2vryTsMQzTX3XRPikhgRwA&branch=master)](https://www.bitrise.io/app/43a23ab54dee9f7e)
+* [Build Status](https://www.bitrise.io/app/43a23ab54dee9f7e/status.svg?token=2vryTsMQzTX3XRPikhgRwA&branch=master)](https://www.bitrise.io/app/43a23ab54dee9f7e)
```

Comment:
is this intentional?
In my opinion before the change it looks better.
actual README:
![grafik](https://user-images.githubusercontent.com/53038537/117507195-0a1c3a80-af87-11eb-806b-254cb9c46571.png)

your branch:
![grafik](https://user-images.githubusercontent.com/53038537/117507115-f53fa700-af86-11eb-85e5-1902b74a9e96.png)